docs/configuration.md

```diff
@@ -20,8 +20,7 @@ var player = playkit.core.loadPlayer(config);
   network: PKNetworkConfigObject,
   customLabels: PKCustomLabelsConfigObject,
   abr: PKAbrConfigObject,
-  drm: PKDrmConfigObject,
-  dimensions: PKDimensionsConfig
```

Comment:
dod you remove it this on purpose? not used?
With the latest addition by @dan-ziv in #506 I think we can use this as config as well now

Reply:
totally by mistake